### DISCUSSION TITLE
Oh my posh on a WDAC (Windows Defender Application Control) Enforced System

### DISCUSSION THREAD

**Discussion by SebCT** [JanDeDobbeleer/oh-my-posh#2796] [Troubleshoot] [answered]
Hello!

On my workstation with Windows Defender Application Control (WDAC) enabled, which always sets a PowerShell Console Session in Constrained language mode (even as Admin), i get the following error:

<img width="619" alt="image" src="https://user-images.githubusercontent.com/62695489/190540633-24fd53ea-8122-48c0-a5b4-98a6df3ce13d.png">

Any chance to do this in another method for constrained language mode with WDAC? Normally i can sign PowerShell Scripts, so they run always in Full language Mode, but i can't sign the omp.ps1 script because of the EXE Wrapper.

Thanks in advance for any support/help!

**Comment by JanDeDobbeleer** [comment, marked as answer]
There's a fix in #2797

**Reply by SebCT** [reply]
Thank you!

**Comment by SebCT** [comment]
I found another issue with AppLocker/WDAC concerning installation setup, but will open a seperate Q&A for it
